packages/shared/providers.yaml

```diff
@@ -1,3 +1,4 @@
+# yaml-language-server: $schema=./../../scripts/validation/providers/schema.json
```

Comment:
I haven't checked all changes in providers.yaml. Is it possible to verify that all  patterns for example matches existing examples we have in the database?